recipes/tilepie/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "tilepie" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 21c63a35b52800957b0af1f15d33acc80c372db8305c8ecea6bc3335fd34221b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
```

Could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
  noarch: python
```